xp3.sh

```diff
@@ -27,6 +27,6 @@ elif [ "${1:-}" = '-h' ] || [ "${1:-}" = '--help' ]; then usage
 
 else
 
-  ${MORGANA} $@ -xslt-connector=saxon12-3 -indent-errors
+  ${MORGANA} -config=lib\morgana-config.xml $@ -indent-errors
```

Comment:
Since this shell script is in batch and "platform-specific" (as opposed the sibling Windows `xp3.bat` file), the proper path separator has to be `/` whether `bash` is running natively on Linux/Unix (macOS) or Windows in the msysgit bash emulator.

```suggestion
  ${MORGANA} -config=lib/morgana-config.xml $@ -indent-errors
```